server/webhook.go

```diff
@@ -691,17 +722,47 @@ func (s *Server) prepareIssueFromPushEventDDL(ctx context.Context, repo *api.Rep
 			)
 			return nil
 		}
+		for _, task := range taskList {
```

Comment:
This only handles the none tenant-mode case. I guess we can add a TODO here.

Shall we use a SQL query to find the relevant task using JSONB?

Reply:
Done.